### PR TITLE
[one-cmds] Revise one-optimize as python script

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit revises one-optimize as python script.

Related : #4278
Draft: #4758
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
CC @struss , @llFreetimell , @jinevening

**Review by seanshpark — DISMISSED** [review]
LGTM

File: compiler/one-cmds/one-optimize [hunk 1]
```diff
@@ -14,155 +19,123 @@
...
+    with open(logfile_path, 'wb') as f:
+        # make a command to optimize circle model
+        circle2circle_path = os.path.join(dir_path, 'circle2circle')
+        circle2circle_cmd = _utils._make_circle2circle_cmd(args, circle2circle_path,
+                                                           getattr(args, 'input_path'),
+                                                           getattr(args, 'output_path'))
+
+        f.write((' '.join(circle2circle_cmd) + '\n').encode())
+
+        # convert tflite to circle
```
Comment (jinevening) [inline compiler/one-cmds/one-optimize:112]:
```suggestion
        # convert circle to circle
```

**Review by jinevening — APPROVED** [review]

**Comment by mhs4670go** [comment]
@seanshpark @jinevening PTAL:)

**Review by jinevening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**